gvm/protocols/gmpv7/gmpv7.py

```diff
@@ -4807,20 +4808,20 @@ def modify_config_set_family_selection(
         for family in families:
             _xmlfamily = _xmlfamsel.add_element("family")
             _xmlfamily.add_element("name", family[0])
-            _xmlfamily.add_element("all", "1")
 
-            if len(family) < 2:
-                raise InvalidArgument(
-                    "Family must have boolean as second argument."
-                )
+            if len(family) != 3:
+                raise InvalidArgument("Family must be a tuple of 3. (str, bool, bool)")
```

Comment:
Sorry messed this up. The line is too long. :O

```suggestion
                raise InvalidArgument(
                    "Family must be a tuple of 3. (str, bool, bool)"
                )
```